python-sdk/nuscenes/eval/lidarseg/utils.py

```diff
@@ -208,6 +213,29 @@ def get_coarse2idx(self) -> Dict[str, int]:
                 'manmade': 15,
                 'vegetation': 16}
 
+    def get_coarse2color(self) -> Dict[str, Tuple[int]]:
+        """
+        Returns the mapping from the coarse class names to the colors in RGB.
+        :return: A dictionary containing the mapping from the coarse class names to the colors in RGB.
+        """
+        return {self.ignore_class['name']: (0, 0, 0),  # Black.
+                'barrier': (112, 128, 144),  # Slategrey
+                'bicycle': (220, 20, 60),  # Crimson
+                'bus': (255, 127, 80),  # Coral
+                'car': (255, 158, 0),  # Orange
+                'construction_vehicle': (233, 150, 70),  # Darksalmon
+                'motorcycle': (255, 61, 99),  # Red
+                'pedestrian': (0, 0, 230),  # Blue
+                'traffic_cone': (47, 79, 79),  # Darkslategrey
+                'trailer': (255, 140, 0),  # Darkorange
+                'truck': (255, 99, 71),  # Tomato
+                'driveable_surface': (0, 207, 191),  # nuTonomy green
+                'other_flat': (175, 0, 75),
+                'sidewalk': (75, 0, 75),
+                'terrain': (112, 180, 60),
+                'manmade': (222, 184, 135),  # Burlywood
+                'vegetation': (0, 175, 0)}  # Green
```

Comment:
Can we use them from the original definition, rather than copying them here?

Reply:
Unfortunately, when we merge the classes, we need to define what colors the merged classes take